### ISSUE TITLE
Multiplatform when app name has a space and specify path

### ISSUE THREAD

**Issue by Reedyuk** [realm/realm-kotlin#1194]
I have an app where i am embedding an existing .realm file(words.realm) and my app name is "word check".
When i pass this path from ios:
```
Bundle.main.url(forResource: "words", withExtension: ".realm")!.path .replacing("words.realm", with: "")
```
It will produce a path like this(using sim, but same applies for device):
```
Users/reedyuk/Library/Developer/CoreSimulator/Devices/2167F46A-B454-49FF-9466-7ACD5579EAEC/data/Containers/Bundle/Application/8A60DDAF-253D-473D-B947-2B54C7A40C12/Word Check.app/
```
but the error output is:
```
Users/reedyuk/Library/Developer/CoreSimulator/Devices/2167F46A-B454-49FF-9466-7ACD5579EAEC/data/Containers/Bundle/Application/8A60DDAF-253D-473D-B947-2B54C7A40C12/Word%20Check.app/
```
But i would assume its failing because its treating the path as a url.

**Comment by rorbech** [comment]
Yes, seems like the implementation is somehow causing issues on iOS even thought it is using the same API 🤔  Thanks for 
the report.

**Comment by Reedyuk** [comment]
Many thanks for the fix, when will it be released? I see its been merged in but no release?

**Comment by rorbech** [comment]
@Reedyuk I don't have the ETA for a numbered release, but it is already available in `1.5.2-SNAPSHOT` already published. You can read how to use that in https://github.com/realm/realm-kotlin#using-snapshots

**Comment by rorbech** [comment]
The fix is now also available in `1.5.2`
